Source/encrypt.cpp

```diff
@@ -4,10 +4,9 @@
  * Implementation of functions for compression and decompressing MPQ data.
  */
 #include <memory>
+#include <SDL.h>

```

Comment:
This include should not be necessary here, what is it for?

Reply:
`SDL_SwapLE32` is being utilized.